CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.5.2:
```

Comment:
This should be `## Master`. I've pushed an empty top-level section to our `master` branch, so you should be able to rebase it over that.